### DISCUSSION TITLE
Fix for Flatpak / Portals Notifications when using `xdg-desktop-portal-gnome`

### DISCUSSION THREAD

**Discussion by cyrneko** [YaLTeR/niri#902] [General]
I noticed that when using the GNOME Portal, Flatpak (and other portal-)notifications would not come through, presumably as the portal was trying to use GNOME Shell which...we don't have

So after looking up the correct string to use I've added this to `/usr/share/xdg-desktop-portal/niri-portals.conf`:
```
org.freedesktop.impl.portal.Notification=gtk;
```

So, using the generic GTK portal for notifications specifically makes it correctly forward notifications to, for instance, `swaync` or `dunst`

Leaving this here for anyone else that might need it, might also be interesting to change the default configuration to `gtk,gnome` or something else? :thinking:

**Comment by YaLTeR** [comment]
Yeah, I also fixed it on main a while ago: https://github.com/YaLTeR/niri/commit/82e30246c1c86b6d20861ede98c8fac4df141aff

**Reply by cyrneko** [reply]
oh neat!
